### ISSUE TITLE
i don't think it is a good idea to opensource this

### ISSUE THREAD

**Issue by alagiz** [Luna-Station-88/terra-luna-catcher#1]
more and more people are getting scammed daily, getting their luna unbonded, and scammers can use this code and instructions to beat users in withdrawing it once it is unbonded.
just saying.

**Comment by Georgetsagg** [comment]
This bot is my only hope to save my unbounded from scammers LUNA.. I'm sure that they also have the bot set up.. or maybe a better one

**Comment by glitzkri3g** [comment]
Would love to combine forces if you two are working on similar projects to rescue your Luna. I got phished and still have a lot of Luna tied up in delegation.

**Comment by Georgetsagg** [comment]
Yes please contact me at Discord. georgetsa#3123  i have 7 days left..

**Comment by pattymfish** [comment]
Hi guys I have same issue my terra wallet was compromised and I have quite a bit of LUNA sitting in delegation and no idea how to rescue it. any chance I can get some help as well.

**Comment by romansvet** [comment]
> Hi guys I have same issue my terra wallet was compromised and I have quite a bit of LUNA sitting in delegation and no idea how to rescue it. any chance I can get some help as well.

Try to contact Joe |  High Stakes#0880 at Discord Chat https://discord.gg/95tHAYVx
He has more options than this code gives. And he does not give any guarantees either, but sometimes he manages to return the funds.

**Comment by romansvet** [comment]
> more and more people are getting scammed daily, getting their luna unbonded, and scammers can use this code and instructions to beat users in withdrawing it once it is unbonded. just saying.

this code is very simple, any scammer can do much better, so this is just a chance for those who can't write code to get their money back.
